File: Week_0.1/q16.java
import java.util.Scanner;

public class q16 {
    public static void main(String arg[]) {
        System.out.print("Enter a number : ");
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();

        for( int i = 2; i < num; i++) {
            if( num % i == 0) {
                System.out.println("Entered number is not Prime number");
                return;
            }
        }
        System.out.println("Entered number is a Prime number");

    }
}

File: Week_3/q16.js
var readlineSync = require('readline-sync')

let num = readlineSync.question("Enter a number ")

const isPrime = (num) => {
    let l = Math.sqrt(num)
    for (let i = 2; i <= l; i++) {
        if(num % i == 0) {
            console.log("Entered number is not a Prime number")
            return
        }
    }

    if(num > 1) {
        console.log("Entered number is a Prime number")
    }
}

isPrime(num)